internal/server/api.yaml

```diff
@@ -8,6 +8,9 @@ info:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
 
+servers:
+  - url: "/api/image-builder/v1"
+  - url: "/api/image-builder/v1.0"
```

Comment:
nitpick, newline between `servers:` and `paths:`, just to be consistent with the newline between `info:` and `servers:`